server/game/cards/02_SHD/units/PrivateerScyk.ts

```diff
@@ -0,0 +1,23 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Aspect, KeywordName, WildcardLocation } from '../../../core/Constants';
+
+export default class PrivateerScyk extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '0088477218',
+            internalName: 'privateer-scyk',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addConstantAbility({
+            title: 'While you control another Cunning unit, this unit gains Shielded',
+            condition: (context) => context.source.controller.getOtherUnitsInPlayWithAspect(context.source, Aspect.Cunning).length > 0,
```

Comment:
isAspectInPlay

Reply:
I'm not sure the shorter name is clear. Do we have any units that check their own aspect(s)?

Reply:
Oh that's a new method Val just added, it includes an `ignoreUnit` parameter to filter out a selected card for exactly that reason

Reply:
Oh wait I'm dumb, yeah I just realized we have duplicates of this API.

Reply:
Yeah just putting it together now that there's a bit of redundancy, but we can let it be for now and see if there are distinct use cases over time.